### PR TITLE
Add Login migration prerequisites in Target selection page

### PR DESCRIPTION
**Pull request by Bhavikshah123**
This PR is for communicating clearly to customers the prerequisites for login migration on 1st page of login migration wizard

image

### CODE REVIEW

File: extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts [hunk 1]
```diff
@@ -97,10 +90,67 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
...
+				}
+			}).component();
+
+		const preRequisiteListInfoBox = this._view.modelBuilder.infoBox()
+			.withProps({
+				style: 'information',
+				text: constants.LOGIN_MIGRATIONS_PRE_REQ_INFO,
+				CSSStyles: { ...styles.BODY_CSS },
+				links: [
+					{ text: constants.VIEW_TUTORIAL, url: 'https://learn.microsoft.com/en-us/azure/dms/tutorial-login-migration-ads' },
```
Comment (stuti149) [inline extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts:132]: Nit: Define url as a constant

**Review by stuti149 — APPROVED** [review]

File: extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts [hunk 2]
```diff
@@ -111,12 +161,17 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
 				CSSStyles: { 'padding-top': '0' }
 			}).component();
 
+		tasksContainer.addItem(form, {});
```
Comment (sahuroshan) [inline extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts:164]: nit : empty second parameter might not be required

File: extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts (hunk 1 above)
Comment (Bhavikshah123) [reply extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts:132]: This is the pattern followed throughout the code where links are added in infobox, so keeping it the same for consistency.

File: extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts (hunk 2 above)
Comment (Bhavikshah123) [reply extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts:164]: This is the pattern followed throughout the code where we have multiple items in the same row, also this just adds a empty option item layout which is of no harm, so keeping it the same for consistency.

**Pull request merged**